File: class_tool/src/main/java/com/tempsfteam/class_tool/vo/CourseConnectionVO.java
package com.tempsfteam.class_tool.vo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

/**
 * @author hypocodeemia
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class CourseConnectionVO {
    private Integer courseId;

    private String courseName;

    private String icon;

    private List<SimpleProfessionVO> SimpleProfessionVOList;
}

File: class_tool/src/main/resources/static/src/models/vo/courseConnectionVO.js
export default {
    name: 'courseConnectionVO',
    title: 'CourseConnectionVO',
    fields: [
        { name: 'courseId', label: '课程ID', type: 'number' },
        { name: 'courseName', label: '课程名', type: 'string' },
        { name: 'icon', label: '图标', type: 'string' },
        { name: 'SimpleProfessionVOList', label: '专业列表', type: 'array' }
    ]
};